connect/service.js

```diff
@@ -128,7 +128,13 @@ const getRoleMembers = (role) => (
  * @return {Promise}   resolves to the list of user details
  */
 const getUsersById = (ids) => {
-  const query = _.map(ids, (id) => 'userId:' + id).join(' OR ');
+
+  let query = ""
+  if (ids.length>1) {
```

Comment:
Correct me on this, but I thought our coding standard for JS would write this as:
```
if (ids.length > 1) {
```
Let me know if that's off base.